rust_src/src/keymap.rs

```diff
@@ -50,6 +52,33 @@ pub fn keymapp(object: LispObject) -> bool {
     map.is_not_nil()
 }
 
+/// Return the parent map of KEYMAP, or nil if it has none.
+/// We assume that KEYMAP is a valid keymap.
+#[no_mangle]
+pub extern "C" fn keymap_parent(keymap: Lisp_Object, autoload: bool) -> Lisp_Object {
+    let map = keymap_parent_internal(LispObject::from_raw(keymap), autoload);
+    map.to_raw()
+}
+
```

Comment:
Why have the `_internal` function? It looks like if you leave `keymap_parent` returning a `Lisp_Object`. The `lisp_fn` can call it and convert it to `LispObject`.

Reply:
After reviewing it, I think you are right. I guess my initial idea on why I did this is to avoid converting to LispObject back and forth.